src/Illuminate/Filesystem/FilesystemAdapter.php

```diff
@@ -610,6 +620,20 @@ protected function concatPathToUrl($url, $path)
         return rtrim($url, '/').'/'.ltrim($path, '/');
     }
 
+    /**
+     * Replace base parts of UriInterface by values from URL.
+     *
+     * @param  UriInterface  $uri
+     * @param  string  $url
+     * @return UriInterface
```

Comment:
bad phpdoc

Reply:
@GrahamCampbell this was fixed after merging the PR